src/microbe_stage/components/OrganelleContainer.cs

```diff
@@ -588,6 +601,7 @@ public static void CalculateOrganelleLayoutStatistics(this ref OrganelleContaine
     /// </summary>
     public static void FetchLayoutOrganelleComponents(this ref OrganelleContainer container)
     {
+        container.MucocystCount = 0;
```

Comment:
Setup for this variable should be like for the agent vacuole count rather than the slime jets list. (so this is the wrong method to initialize this).